pandas/tests/resample/test_deprecated.py

```diff
@@ -0,0 +1,264 @@
+from datetime import datetime, timedelta
+
+import numpy as np
```

Comment:
can you ensure that we have replicated the loffset / base tests using offset / orgin. (this can be a followup as well). The idea is that when we enforce the deprecation we can simply remove the depredated tests and don't need to add any.

pls lmk either way.

Reply:
I have replicated the offset/origin tests:

- https://github.com/pandas-dev/pandas/pull/31809/files/689180c70a7833042a771795d78f8c667f5d3e54#diff-dd427a6bbdfef7e333ac320adb15614dR738
- https://github.com/pandas-dev/pandas/pull/31809/files/689180c70a7833042a771795d78f8c667f5d3e54#diff-dd427a6bbdfef7e333ac320adb15614dR738
- https://github.com/pandas-dev/pandas/pull/31809/files/689180c70a7833042a771795d78f8c667f5d3e54#diff-dd427a6bbdfef7e333ac320adb15614dR738
- https://github.com/pandas-dev/pandas/pull/31809/files/689180c70a7833042a771795d78f8c667f5d3e54#diff-967cd812d0942138129aacf0f9163604R82

however, I did not replicated the `loffset` argument since now it is up to the user to add the `loffset` to a `dataframe.index`

Reply:
so all tests that were above are now here?

> however, I did not replicated the loffset argument since now it is up to the user to add the loffset to a dataframe.index

what does this mean?

Reply:
> so all tests that were above are now here?

Yes, all deprecated tests have been moved into `test_deprecated.py`.
When you told me to remove some deprecated tests I moved them instead into `test_deprecated.py`: https://github.com/pandas-dev/pandas/pull/31809#discussion_r379711472

> > however, I did not replicated the loffset argument since now it is up to the user to add the loffset to a dataframe.index
> 
> what does this mean?

I have just moved the `loffset` tests into `test_deprecated.py` since neither of the two new arguments `origin` or `offset` really replace the now deprecated `loffset` argument.

`loffset` being just a timedelta that we add at the end of the resample, this behavior is just not handled with the new `origin` or `offset`.

To explain further, here a small notebook snippet (pretty much copy/paste from the new doc):

```python
>>> import pandas as pd
>>> import numpy as np
>>> from pandas.tseries.frequencies import to_offset
```

Setting up the timeseries and define a `my_loffset` for convenience
```python
>>> start, end = "2000-10-01 23:30:00", "2000-10-02 00:30:00"
>>> my_loffset = "19min"
>>> rng = pd.date_range(start, end, freq="7min")
>>> ts = pd.Series(np.arange(len(rng)) * 3, index=rng)
>>> ts
```
```
2000-10-01 23:30:00     0
2000-10-01 23:37:00     3
2000-10-01 23:44:00     6
2000-10-01 23:51:00     9
2000-10-01 23:58:00    12
2000-10-02 00:05:00    15
2000-10-02 00:12:00    18
2000-10-02 00:19:00    21
2000-10-02 00:26:00    24
Freq: 7T, dtype: int64
```
The old way:
```python
>>> ts_out = ts.resample("17min", loffset=my_loffset).sum()
>>> ts_out
```
```
2000-10-01 23:33:00     0
2000-10-01 23:50:00     9
2000-10-02 00:07:00    21
2000-10-02 00:24:00    54
2000-10-02 00:41:00    24
Freq: 17T, dtype: int64
```

The way to replace `loffset` after v1.1.0:
```python
>>> ts_out = ts.resample("17min").sum()
>>> ts_out.index = ts_out.index + to_offset(my_loffset)
>>> ts_out
```
```
2000-10-01 23:33:00     0
2000-10-01 23:50:00     9
2000-10-02 00:07:00    21
2000-10-02 00:24:00    54
2000-10-02 00:41:00    24
Freq: 17T, dtype: int64
```